gafaelfawr/templates/ingress.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+  name: {{ template "helpers.fullname" . }}-ingress
+spec:
+  rules:
+{{ if .Values.ingress.host }}
+  - host: {{ .Values.ingress.host }}
+    http:
+{{ else }}
+  - http:
+{{ end }}
+      paths:
+      - path: /analyze
```

Comment:
Should this be /auth/analyze?  I'm a bit confused, as I see /auth but I thought all of these would start with /auth - or are we making a /login on the root now?

Reply:
Oh, right, yes, it should be `/auth/analyze`, which in turn means that I can delete this block and change the `pathType` setting to `Prefix`. Done.

I was going to add `/login` and `/logout` at the root level because... I dunno, they felt special. If you think that's a bad idea, I can go back to mounting everything under `/auth` and letting the URLs be a little bit longer.

Reply:
I think it's a good idea just was mostly confused at the confluence of these two points